controllers/api/eventRoutes.js

```diff
@@ -1,38 +1,176 @@
 const router = require('express').Router();
-const { Event } = require('../../models');
+const { Event, User, Task } = require('../../models');
```

Comment:
I think that each of the route files only deals with their specific model so the eventRoutes only requires the Event model.  I am sure loading the others is okay but not necessary (and isn't the way it was done in the mini-project (which I will reference often))